.github/workflows/battery_plus.yaml

```diff
@@ -117,19 +117,40 @@ jobs:
         run: ./.github/workflows/scripts/build-examples.sh linux ./lib/main.dart
 
   windows:
-    if: false # Disabled until Windows CI scripts are done
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
     timeout-minutes: 30
     steps:
       - name: "Checkout repository"
         uses: actions/checkout@v2
       - name: "Install Flutter"
+        shell: bash
         run: ./.github/workflows/scripts/install-flutter.sh stable
       - name: "Install Tools"
+        shell: bash
         run: ./.github/workflows/scripts/install-tools.sh
       - name: "Build Example"
+        shell: bash
         run: ./.github/workflows/scripts/build-examples.sh windows ./lib/main.dart
 
+  windows_integration_test:
+    runs-on: windows-latest
+    timeout-minutes: 30
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v2
+      - name: "Install Flutter"
+        shell: bash
+        run: ./.github/workflows/scripts/install-flutter.sh stable
+      - name: "Install Tools"
+        shell: bash
+        run: ./.github/workflows/scripts/install-tools.sh
+      - name: "Bootstrap Workspace"
+        shell: bash
+        run: flutter pub global run melos bootstrap
```

Comment:
[see](https://github.com/invertase/melos/issues/241). maybe, In windows it can't set global env var.